solr/core/src/test/org/apache/solr/search/TestExtendedDismaxParser.java

```diff
@@ -1794,6 +1798,37 @@ public void testOperatorsAndMultiWordSynonyms() throws Exception {
     );
   }
 
+  public void testOverlapTermScoringQueries() throws Exception {
+    ModifiableSolrParams edismaxParams = new ModifiableSolrParams();
+    edismaxParams.add("qf", "t_pick_best_foo");
+
+    QParser qParser = QParser.getParser("tabby", "edismax", req(edismaxParams));
+    Query q = qParser.getQuery();
+    assertEquals("+((t_pick_best_foo:tabbi | t_pick_best_foo:cat | t_pick_best_foo:felin | t_pick_best_foo:anim))", q.toString());
+
+    edismaxParams = new ModifiableSolrParams();
```

Comment:
Solr tests have a `params()` method which is much more concise and doesn't pollute the variable namespace unnecessarily